<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>829</width>
    <height>538</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="dashboard_tab">
       <attribute name="title">
        <string>DashBoard</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_9">
          <property name="font">
           <font>
            <pointsize>14</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Actions</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_3">
          <property name="text">
           <string>New Book</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_10">
          <property name="font">
           <font>
            <pointsize>14</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Books</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_19">
            <item>
             <widget class="QLabel" name="label_11">
              <property name="font">
               <font>
                <pointsize>14</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Issued Books</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_8">
              <item>
               <widget class="QTableWidget" name="issued_books_table_5">
                <property name="columnCount">
                 <number>8</number>
                </property>
                <column>
                 <property name="text">
                  <string>ID</string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>Name</string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>Author</string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>Description</string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>ISBN</string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>Page_Count</string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>Year</string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>Issued</string>
                 </property>
                </column>
               </widget>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_20">
                <item>
                 <layout class="QVBoxLayout" name="verticalLayout_21">
                  <item>
                   <widget class="QPushButton" name="delete_issued_3">
                    <property name="text">
                     <string>Delete</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="edit_issued_3">
                    <property name="text">
                     <string>Edit</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="refresh_issued_3">
                    <property name="text">
                     <string>Refresh</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <spacer name="verticalSpacer_5">
                  <property name="orientation">
                   <enum>Qt::Vertical</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>20</width>
                    <height>40</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_22">
            <item>
             <widget class="QLabel" name="label_12">
              <property name="font">
               <font>
                <pointsize>14</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Available books</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_9">
              <item>
               <widget class="QTableWidget" name="issued_books_table_6">
                <property name="columnCount">
                 <number>8</number>
                </property>
                <column>
                 <property name="text">
                  <string>ID</string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>Name</string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>Author</string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>Description</string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>ISBN</string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>Page_Count</string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>Year</string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>Issued</string>
                 </property>
                </column>
               </widget>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_23">
                <item>
                 <layout class="QVBoxLayout" name="verticalLayout_24">
                  <item>
                   <widget class="QPushButton" name="delete_unissued_3">
                    <property name="text">
                     <string>Delete</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="edit_unissued_3">
                    <property name="text">
                     <string>Edit</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="refresh_unissued_3">
                    <property name="text">
                     <string>Refresh</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <spacer name="verticalSpacer_6">
                  <property name="orientation">
                   <enum>Qt::Vertical</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>20</width>
                    <height>40</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="find_tab">
       <attribute name="title">
        <string>Find</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="label">
              <property name="font">
               <font>
                <pointsize>16</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Search By ID</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="search_input"/>
            </item>
            <item>
             <widget class="QPushButton" name="search_button">
              <property name="text">
               <string>Search</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QTableWidget" name="search_table">
            <property name="columnCount">
             <number>8</number>
            </property>
            <column>
             <property name="text">
              <string>ID</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Name</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Author</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Description</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>ISBN</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Page_Count</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Issued</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Year</string>
             </property>
            </column>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="all_book_tab">
       <attribute name="title">
        <string>All Books</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="font">
           <font>
            <pointsize>14</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>All Books</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="refresh_books">
          <property name="text">
           <string>Refresh</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTableWidget" name="all_books">
          <property name="columnCount">
           <number>8</number>
          </property>
          <column>
           <property name="text">
            <string>ID</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Name</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Author</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Description</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>ISBN</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Page_Count</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Issued</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Year</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>829</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
